wallets-testing/test/test.service.ts

```diff
@@ -1,19 +1,29 @@
 import { configService } from '../config';
-import { EthereumNodeService } from '@lidofinance/wallets-testing-nodes';
 import { ETHEREUM_WIDGET_CONFIG } from '@lidofinance/wallets-testing-widgets';
-import { ExtensionService } from '@lidofinance/wallets-testing-extensions';
-import { BrowserContextService, BrowserService } from '../browser';
+import { BrowserService } from '@lidofinance/browser-service';
+import { METAMASK_COMMON_CONFIG } from '@lidofinance/wallets-testing-wallets';
 
 export async function initBrowserService() {
-  const ethereumNodeService = new EthereumNodeService({
-    chainId: ETHEREUM_WIDGET_CONFIG.chainId,
-    rpcUrl: configService.get('RPC_URL'),
-    defaultBalance: 1000,
+  METAMASK_COMMON_CONFIG.LATEST_STABLE_DOWNLOAD_LINK = undefined;
+  return new BrowserService({
+    networkConfig: {
+      chainId: ETHEREUM_WIDGET_CONFIG.chainId,
+      chainName: ETHEREUM_WIDGET_CONFIG.chainName,
+      tokenSymbol: ETHEREUM_WIDGET_CONFIG.tokenSymbol,
+      rpcUrl: configService.get('RPC_URL'),
+      scan: 'https://etherscan.io/',
+    },
+    walletConfig: {
+      SECRET_PHRASE: configService.get('WALLET_SECRET_PHRASE'),
+      PASSWORD: configService.get('WALLET_PASSWORD'),
+      COMMON: METAMASK_COMMON_CONFIG,
+      NETWORK_NAME: ETHEREUM_WIDGET_CONFIG.networkName,
```

Comment:
I suppose we can remove this `NETWORK_NAME` in `walletConfig` and use `this.options.networkConfig.chainName` [here](https://github.com/lidofinance/wallets-testing-modules/blob/4e6c020d5afc08efc9c7b83f68a1e7a24602d006/packages/browser-service/src/browser.service.ts#L144C3-L144C64) in so on

Reply:
`NETWORK_NAME` for setup used only by `wallet.setup(networkName)` by some of wallets. I suppose we can try to do with `this.options.networkConfig.chainName`, if not add "networkName" param in `wallets.constants.NETWORKS_CONFIGS` to use is at is.

Reply:
No, because these are different network names.

this.options.networkConfig.chainName might contain 'Ethereum Mainnet', while ETHEREUM_WIDGET_CONFIG.networkName contains 'Ethereum'.

We need ETHEREUM_WIDGET_CONFIG.networkName because Coim98 page require newtworkName = 'Ethereum'

Reply:
Got you.

I suppose we can use `wallets.constants.NETWORK_CONFIG.Mainnet.Ethereum` and add  `networkName` property. 

then use 
```
    networkConfig: {
      ...NETWORKS_CONFIG.Mainnet.ETHEREUM,
      rpcUrl: configService.get('RPC_URL'),
    }
    ```
    and remove `NETWORK_NAME: ETHEREUM_WIDGET_CONFIG.networkName,`
 and use `await extensionWalletPage.setup(this.options.networkConfig.networkName)`

Reply:
ETHEREUM_WIDGET_CONFIG will remove by this ticket https://linear.app/lidofi/issue/QA-3566/remove-widget-package-and-move-to-wallet-testing-directory